tests/gis_tests/test_geoip2.py

```diff
@@ -130,23 +130,9 @@ def test04_city(self, gethostbyname):
 
             geom = g.geos(query)
             self.assertIsInstance(geom, GEOSGeometry)
-            lon, lat = (-95.4010, 29.7079)
-            lat_lon = g.lat_lon(query)
-            lat_lon = (lat_lon[1], lat_lon[0])
-            for tup in (geom.tuple, g.coords(query), g.lon_lat(query), lat_lon):
-                self.assertAlmostEqual(lon, tup[0], 4)
-                self.assertAlmostEqual(lat, tup[1], 4)
-
-    @mock.patch('socket.gethostbyname')
-    def test05_unicode_response(self, gethostbyname):
-        "GeoIP strings should be properly encoded (#16553)."
-        gethostbyname.return_value = '194.27.42.76'
-        g = GeoIP2()
-        d = g.city('nigde.edu.tr')
-        self.assertEqual('Niğde', d['city'])
-        d = g.country('200.26.205.1')
-        # Some databases have only unaccented countries
-        self.assertIn(d['country_name'], ('Curaçao', 'Curacao'))
+            lat, lon = g.lat_lon(query)
+            self.assertTrue(lat)
```

Comment:
I think in the first version of the patch you had something like `assertIsInstance(lat, float)` which seemed better to me.